WordPress/Classes/Services/PeopleService.swift

```diff
@@ -395,12 +395,8 @@ extension PeopleService {
     ///   - onComplete: A completion block that is called after changes are saved to core data.
     ///
     func merge(remoteInvites: [RemoteInviteLink], for siteID: Int, onComplete: @escaping (() -> Void)) {
-        let context = ContextManager.shared.newDerivedContext()
-        context.perform {
-            guard let blog = try? Blog.lookup(withID: siteID, in: context) else {
-                DispatchQueue.main.async {
-                    onComplete()
-                }
+        ContextManager.shared.performAndSave { context in
+            guard let blog = try Blog.lookup(withID: siteID, in: context) else {
                 return
             }
```

Comment:
I was like, _hang on, where did the completion call go?_ But, of course, this API has it's own completion parameter later on 👍